doc/obs_decam/index.rst

```diff
@@ -0,0 +1,27 @@
+.. _obs_decam-package:
+
```

Comment:
Delete this whole directory: this doesn't exist in the new system. See the layout here: https://developer.lsst.io/stack/layout-of-doc-directory.html#the-stack-package-template